src/styles/blocks/main_block.scss

```diff
@@ -0,0 +1,23 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: $ver-padding $hor-padding;
+  width: $main-dimension;
+
+  &--side-margins--auto {
+    margin: 0 auto;
+  }
```

Comment:
As for me mix of block and element is better, but ok

Reply:
I can give tag main class="main", and to cards add class "main__cards"